packages/shared-state-node_info/files/etc/uci-defaults/shared-state_node_info_cron

```diff
@@ -1,9 +1,21 @@
 #!/bin/sh
+
 unique_append()
```

Comment:
This function is not used anymore should be removed, also the script should be renamed ;)